addons/sale/data/mail_template_data.xml

```diff
@@ -85,7 +85,7 @@
                 <td width="15%" align="center"><span style="font-weight:bold;">Quantity</span></td>
                 <td width="20%" align="right">
                     <span style="font-weight:bold;">
-                        <t t-if="object.company_id.tax_calculation_rounding_method == 'round_per_line'">
+                        <t t-if="object.website_id.show_line_subtotals_tax_selection == 'tax_excluded'">
```

Comment:
`sale` does not depend on `website`.

Reply:
In a `<t t-if="hasattr(object, 'website_id') and object.website_id">` block 🤦

Pay your modularity...

Reply:
Indeed, but that part of the `sale` mail template is encapsulated in a `<t t-if="hasattr(object, 'website_id') and object.website_id">`.

I've been complaining about that template and code for years, but it's still there for now, and we have to deal with it (since there is no way to modify mail template with clean inheritance).